### ISSUE TITLE
Hope to add a knowledge base selection function to the chat interface

### ISSUE THREAD

**Issue by 0902081008** [labring/FastGPT#559]
- [x ] 我已确认目前没有类似 features
- [x ] 我已确认我已升级到最新版本
- [x ] 我已完整查看过项目 README，已确定现有版本无法满足需求
- [x ] 我理解并愿意跟进此 features，协助测试和提供反馈
- [x] 我理解并认可上述内容，并理解项目维护者精力有限，**不遵循规则的 features 可能会被无视或直接关闭**

**功能描述**
希望在聊天界面增加知识库选择功能，这样可以随时切换知识库，询问不同类型的知识。

**应用场景**

**相关示例**

**Comment by c121914yu** [comment]
> Bot detected the issue body's language is not English, translate it automatically. 👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿         
----    
- [x ] I have confirmed that there are currently no similar features
- [x ] I have confirmed that I have upgraded to the latest version
- [x ] I have fully reviewed the project README and determined that the existing version cannot meet the needs.
- [x ] I understand and am willing to follow up on this feature, assist with testing and provide feedback
- [x] I understand and agree with the above content, and understand that project maintainers have limited energy. **Features that do not follow the rules may be ignored or closed directly**

**Function description**
I hope to add a knowledge base selection function to the chat interface, so that I can switch knowledge bases at any time and ask for different types of knowledge.

**Application Scenario**

**Related Examples**
